ecl/hqlcpp/hqlcpp.ipp

```diff
@@ -2146,19 +2146,23 @@ protected:
     SummaryMap summaries[(int) SummaryType::NumItems];
     void noteSummaryInfo(const char *name, SummaryType type, bool isOpt, bool isSigned)
     {
-        if (type != SummaryType::None)
-        {
-            SummaryMap &map = summaries[(int) type];
-            SummaryFlags flags = SummaryFlags::None;
-            if (isOpt)
-                flags |= SummaryFlags::IsOpt;
-            if (isSigned)
-                flags |= SummaryFlags::IsSigned;
-            if (map.find(name) == map.end())
-                map[name] = flags;
-            else
-                map[name] = map[name] & flags;
-        }
+        if (type == SummaryType::None)
+            return;
+        //Spill files are meaningless in roxie, and no current benefit in recording them for hthor/thor
```

Comment:
are SummaryType::JobTemp's useful?

Reply:
They might be - in that they are written to persistent storage, so they are potentially noteworthy.  We can always remove recording them at a later date, and they will not take up a significant amount of space.